Examples/NoModules/Appendix A/ExA_04/ExA_04.cpp

```diff
@@ -0,0 +1,21 @@
+// Demonstrating assertions
+#include <iostream>
+#include <format>
+#include <cassert> 
+
+// Uncomment to disable the assertion in main()
+//#define NDEBUG
```

Comment:
Even when uncommenting, the assert is still triggered.
You need to move this line to before you include cassert.